src/main/scala/ai/eto/rikai/sql/model/Registry.scala

```diff
@@ -57,7 +79,8 @@ private[rikai] object Registry {
     */
   val DEFAULT_REGISTRIES = Map(
     "rikai.sql.ml.registry.file.impl" -> "ai.eto.rikai.sql.model.fs.FileSystemRegistry",
-    "rikai.sql.ml.registry.mlflow.impl" -> "ai.eto.rikai.sql.model.mlflow.MlflowRegistry"
+    "rikai.sql.ml.registry.mlflow.impl" -> "ai.eto.rikai.sql.model.mlflow.MlflowRegistry",
```

Comment:
This is amazing.  How much work is there to make Rikai to support multiple model registries?

Reply:
I took about 1d to create this pull request for the torchhub registry.

Reply:
For multiple model registries, I created a issue: https://github.com/eto-ai/rikai/issues/377

We need to design syntax first.

Reply:
For multiple model registries, I created a issue: #377

We need to design syntax first.